### PR TITLE
sorter/leveldb(ticdc): fix input regression panic

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Input events may regress, so skip panic. 

Close https://github.com/pingcap/tiflow/issues/3969

### Check List 

Tests 

 - Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by liuzix — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

File: cdc/sorter/leveldb/table_sorter.go [hunk 1]
```diff
@@ -138,9 +138,10 @@ func (ls *Sorter) wait(
 	inputCount, kvEventCount, resolvedEventCount := 0, 0, 0
 	appendInputEvent := func(ev *model.PolymorphicEvent) {
 		if ls.lastSentResolvedTs != 0 && ev.CRTs < ls.lastSentResolvedTs {
-			log.Panic("commit ts < resolved ts",
+			log.Warn("commit ts < resolved ts",
 				zap.Uint64("lastSentResolvedTs", ls.lastSentResolvedTs),
 				zap.Any("event", ev), zap.Uint64("regionID", ev.RegionID()))
+			return
```
Comment (amyangfei) [inline cdc/sorter/leveldb/table_sorter.go:144]: Please add some comments about why this can happen, and what does regression affect here.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**